eng/pipelines/templates/steps/build-test.yml

```diff
@@ -66,6 +66,15 @@ steps:
       displayName: 'Dump Test Proxy logs'
       condition: succeededOrFailed()
 
+  - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+    displayName: 'Generate BOM'
+    inputs:
+      BuildDropPath: ${{ parameters.ServiceDirectory }}
+
+  - publish: ${{ parameters.ServiceDirectory }}/_manifest
+    displayName: 'Publish BOM Manifest'
+    artifact: 'manifest'
```

Comment:
Should we be consistent and call all the uploads manifest for the other languages as well?

Reply:
Yessir. I'm actually adjusting these in the original PRs to match.